Entities/ACDropShip.cpp

```diff
@@ -152,58 +152,40 @@ int ACDropShip::ReadProperty(std::string propName, Reader &reader)
         delete m_pRThruster;
         m_pRThruster = new AEmitter;
         reader >> m_pRThruster;
-        m_pRThruster->Attach(this);
```

Comment:
Why did you remove all the attach calls in the various Actor subclass ReadProperty stuff?

Reply:
We're calling Attach() from AddAttachable() when we're doing deepcopy Create.
So it just looked unnecessary to call it twice.

Reply:
Okay, that's what I figured. I think you're right and if it appears to work that's fine then.

Reply:
It works just fine because we aren't/can't use the original read-from-ini entity.
If we were/could that would be an issue because you'd end up with an entity that has parentless attachables.

Reply:
Yeah, that sounds reasonable. Cool.